src/main.go

```diff
@@ -128,11 +133,60 @@ func (app *App) parse(b []byte) ([]byte, error, error) {
 	return pc, errParse, err
 }
 
+func (app *App) parseRemoteURL(urlString string) ([]byte, error, error) {
+	log.Infof("called parseRemoteURL() url: %s", urlString)
+	p := publiccode.NewParser()
+	errParse := p.ParseRemoteFile(urlString)
+	pc, err := p.ToYAML()
+
+	return pc, errParse, err
+}
+
+func (app *App) validateRemoteURL(w http.ResponseWriter, r *http.Request) {
+	log.Info("called validateFromURL()")
+	setupResponse(&w, r)
+	if (*r).Method == "OPTIONS" {
+		return
+	}
+	// Getting vars from parameters
+	vars := mux.Vars(r)
+	urlString := vars["url"]
+	if urlString == "" {
+		return
+	}
+
+	//parsing
+	pc, errParse, errConverting := app.parseRemoteURL(urlString)
+
+	if errConverting != nil {
+		log.Errorf("Error converting: %v", errConverting)
```

Comment:
Are you just logging the error? Nothing is returned?

Reply:
👍